src/utils/queryEventDriver.ts

```
@@ -0,0 +1,34 @@
+      registerActivity(`query_event:${next.value.type}`)
+      onEvent(next.value)
+    }
+  } catch (error) {
+    failed = true
+    throw error
+  } finally {
+    if (!generatorDone) {
+      try {
+        await queryGenerator.return(undefined as never)
+      } catch (cleanupError) {
+        if (!failed) throw cleanupError
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)